Library/Homebrew/cmd/log.rb

```diff
@@ -44,6 +44,8 @@ def log
       git_log HOMEBREW_REPOSITORY, args: args
     else
       path = Formulary.path(args.named.first)
+      warn "#{path} didn't exist. It was deleted or not in homebrew/homebrew-core originally" unless path.exist?
```

Comment:
```suggestion
      opoo "#{args.named.first} does not currently exist in homebrew/homebrew-core" unless path.exist?
```

I think I might like something like this better. Also, use `opoo` instead of `warn` to format the warning more nicely.